package.json

```diff
@@ -18,11 +18,14 @@
     "@supabase/supabase-js": "^2.39.2",
     "@t3-oss/env-nextjs": "^0.7.1",
     "@vercel/analytics": "^1.2.2",
+    "copy-text-to-clipboard": "^3.2.0",
     "dayjs": "^1.11.10",
     "next": "^14.0.4",
     "react": "18.2.0",
+    "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.49.2",
+    "sonner": "^1.4.41",
```

Comment:
wondering why you chose sonner over react-hot-toast?

Reply:
easy customization